### PR TITLE
Cut 0.8.0

### PR DESCRIPTION
**Pull request by ecordell**

### CODE REVIEW

File: manifests/0000_30_07-certified-operators.configmap.yaml [hunk 1]
```diff
@@ -1152,46 +1166,40 @@ data:
...
-      
           - name: mongodbshardedclusters.mongodb.com
             description: MongoDB Sharded Cluster Deployment
             displayName: MongoDB Sharded Cluster
             group: mongodb.com
             kind: MongoDbShardedCluster
             version: v1
-      
             resources:
+            # FIXME(alecmerdler): These are probably supposed to be different resources for the different CRD configurations
```
Comment (njhale) [inline manifests/0000_30_07-certified-operators.configmap.yaml]: Do we want this in our release artifacts?

Comment (ecordell) [reply manifests/0000_30_07-certified-operators.configmap.yaml]: I think so?

**Pull request merged**